src/store/metadata/stub/content/editions/versions/domestic_energy_performance_certi_existing_dwellings_by_eng_eff_rating.json

```diff
@@ -0,0 +1,65 @@
+{
+    "@context": "https://staging.idpd.uk/#ns",
+    "@id": "https://staging.idpd.uk/datasets/domestic_energy_performance_certi_existing_dwellings_by_eng_eff_rating_2023_07/editions/2023-07/versions",
```

Comment:
Change to use globally unique identifier, and change all other instances too.

Reply:
used epcs-for-existing-domestic-properties